example_configs/keycloak_oidc/policy/rbac/rbac.rego

```diff
@@ -0,0 +1,98 @@
+package rbac
+
+import data.token
+
+public_tag := {"public"}
+
+admin_tag := "facility_admin"
+tag_permissions := {
+	"beamline_y_user": [
+		"read:data",
+		"read:metadata",
+	],
+	admin_tag: [
+		"read:data",
+		"read:metadata",
+		"write:data",
+		"write:metadata",
+		"create",
+		"register",
+		"delete:node",
+		"delete:revision"
+	],
+	"beamline_x_user": [
+		"read:data",
+		"read:metadata",
+	],
+	"public": [
+		"read:data",
+		"write:data",
```

Comment:
This tag configures what unauthenticated requests can do. Generally I would expect those would never be allowed to write (or create, register, delete).